apple/Handlers/RNLongPressHandler.m

```diff
@@ -94,6 +107,83 @@ - (void)touchesCancelled:(NSSet<RNGHUITouch *> *)touches withEvent:(UIEvent *)ev
   [self reset];
 }
 
+#else
+- (void)mouseDown:(NSEvent *)event
+{
+  self.state = NSGestureRecognizerStateBegan;
+  startTime = CACurrentMediaTime();
+
+  [_gestureHandler.pointerTracker touchesBegan:[NSSet setWithObject:event] withEvent:event];
+
+  _initPosition = [self locationInView:self.view];
+
+  __weak typeof(self) weakSelf = self;
+
+  block = dispatch_block_create(0, ^{
+    __strong typeof(self) strongSelf = weakSelf;
+
+    if (strongSelf) {
+      strongSelf.state = NSGestureRecognizerStateChanged;
+      strongSelf.state = NSGestureRecognizerStateEnded;
```

Comment:
I'm not sure whether long press should end immediately after activation. I think on other platforms it stays active until the pointer is lifted.

Reply:
Yes, you're right. Changed in [85fa302](https://github.com/software-mansion/react-native-gesture-handler/pull/3014/commits/85fa302b7a8c4f6b177509c2d5abbc689176fb7d)